app/src/main/java/org/schabi/newpipe/player/event/PlayerGestureListener.java

```diff
@@ -69,26 +69,22 @@ public void onSingleTap(@NotNull final MainPlayer.PlayerType playerType) {
         if (DEBUG) {
             Log.d(TAG, "onSingleTap called with playerType = [" + player.getPlayerType() + "]");
         }
-        if (playerType == MainPlayer.PlayerType.POPUP) {
 
-            if (player.isControlsVisible()) {
-                player.hideControls(100, 100);
-            } else {
-                player.getPlayPauseButton().requestFocus();
-                player.showControlsThenHide();
-            }
+        if (player.isControlsVisible()) {
 
-        } else /* playerType == MainPlayer.PlayerType.VIDEO */ {
+            // -- Controls visible --
```

Comment:
This comment and the newlines sorrounding it are unneeded, the argument of the `if` says the exact same thing and the comment below also clears any possible leftover doubt